### ISSUE TITLE
Used Sollet to convert USDT to erc20 via metamask - Still cannot find where the money is!

### ISSUE THREAD

**Issue by tory2777** [project-serum/spl-token-wallet#413]
I knew so many posts out there with this issue...SO any SINCERE solutions is really appreciate here and no SCAM pls. 
it been 2 days and i still cnnot figure out where to get those tokens back to my metamask? 

Where is SOLLET support that i can rely on ?

**Comment by armaniferrante** [comment]
Please join the discord. There are lots of people there to help https://discord.gg/Wg3FZ6jX.

**Comment by tory2777** [comment]
> Please join the discord. There are lots of people there to help https://discord.gg/Wg3FZ6jX.

thx man. Can i honestly ask why so many people had similar issues but no clear solutions share out . Can it be solved?

**Comment by armaniferrante** [comment]
> > Please join the discord. There are lots of people there to help https://discord.gg/Wg3FZ6jX.
> 
> thx man. Can i honestly ask why so many people had similar issues but no clear solutions share out . Can it be solved?

Yes. All such issues have been solved. Please share further details in discord.

**Comment by DCnorthbeach** [comment]
Hi i have had a similar issue! Can someone please help? Invite link to discord is expired. Please

**Comment by armaniferrante** [comment]
https://discord.gg/fRtaCmYSdw
